build_docs.pl

```diff
@@ -726,12 +726,7 @@ sub push_changes {
 #===================================
 sub init_env {
 #===================================
-    $ENV{SGML_CATALOG_FILES} = $ENV{XML_CATALOG_FILES} = join ' ',
-        file('resources/docbook-xsl-1.78.1/catalog.xml')->absolute,
-        file('resources/docbook-xml-4.5/catalog.xml')->absolute;
-
     $ENV{PATH}
-        = dir('resources/asciidoc-8.6.8/')->absolute
         . ":$FindBin::RealBin:"
```

Comment:
should be a `=` sign here maybe?

Reply:
Yeah! I wonder why the tests passed.....

Reply:
I've entirely dropped the path customizations. I guess we didn't need them!